examples/dialog-modal/js/dialog.js

```diff
@@ -126,9 +126,20 @@ aria.Utils = aria.Utils || {};
    */
   aria.Dialog = function (dialogId, focusAfterClosed, focusFirst) {
     this.dialogNode = document.getElementById(dialogId);
-    var role = this.dialogNode.getAttribute('role');
+    if (this.dialogNode === null) {
+      throw new Error('No element found with id="' + dialogId + '".');
+    }
 
-    if (this.dialogNode === null || !['dialog', 'alertdialog'].includes(role)) {
+    var validRoles = ['dialog', 'alertdialog'];
+    var isDialog = (element.getAttribute('role') || '')
```

Comment:
@sh0ji I think this is failing the build because you're referring to `element`, should that not be `this.dialogNode`?

Reply:
Ha yes, thanks for the quick checks!